### PR TITLE
gitignore: Ignore *.cov metadata

### PR DESCRIPTION
**Pull request by Baisang**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,7 @@ _testmain.go
 
 quilt
 
+*.cov
 *.cov.html
 *.cov.out
```
Comment (ejj) [inline .gitignore]: Do these still happen?

Comment (Baisang) [inline .gitignore]: Yeah, I just made a test directory with `abc.cov`, `abc.cov.html`, and `abc.cov.out`. Adding a `.gitignore` with just `*.cov` only gets rid of `abc.cov`

Comment (ejj) [inline .gitignore]: Oh what I'm saying is, do we still have .cov.out files when we run `make coverage`?

Comment (ejj) [inline .gitignore]: Also, what about *.cov*

Comment (Baisang) [inline .gitignore]: `*.cov*` would get rid of all of them. I'm not opposed to that.

I get `*.cov` and `*.cov.html` files, I'm not sure if others are still getting `*.cov.out`?

**Pull request merged**